docs/source/datasets/index.rst

```diff
@@ -151,6 +151,9 @@ The following columns are included in this dataset:
      - :code:`household_id`
      - Not affected by noise functions; intended use is "ground truth" for testing and validation; consistent across all
        datasets.
+   * - Survey date
+     - :code:`survey_date`
+     - Date on which the survey was conducted; metadata that would not be collected directly; not affected by noise functions.
```

Comment:
I think we need to include the format string, as with DOB below.

Reply:
It looks like the concept model and data don't match:

- The concept model says: "Survey date (stored as a string in MM/DD/YYYY format)".
- In the data, `survey_date` is stored as a `datetime64[ns]` and displays in YYYY-MM-DD format.

I'll go with what's in the data.

Reply:
Oh, wait -- when you say the "data" you mean the pseudopeople output? If the pseudopeople output is a datetime, then it has no inherent format, and we shouldn't say that it has one.

Reply:
It's arguably a bug that the concept model is not being followed here. But at the same time, I do see the downside of making survey_date a string and therefore taking up more memory.

Reply:
Yes, I mean the pseudopeople output (using the sample population). I was about to say, "Stored as a `pandas.Timestamp`, which displays in YYYY-MM-DD format by default." But I could leave out the part about the format...

I think we should keep it in a numeric format if we're not adding noise -- it seems plausible that's how the actual data would be stored.

Reply:
@zmbc I just pushed a commit with the wording above. Let me know if that sounds good or if you think we should change it.

Reply:
Yep that sounds good!

Reply:
Re: it seems plausible that's how the actual data would be stored, I agree, with caveats.

Right now you can configure pseudopeople to apply no noise at all to the DOB column (or any other column) but it still returns it as a string. There is no way for me as a pseudopeople user to specify that date of birth is collected using a structured date data type that would not permit errors like e.g. adding a letter to it. I remember talking through this, and our idea was that such a user would just turn off the noise _types_ they don't want, and if that makes their dates (or integers, etc) inherently perfectly parse-able, we leave it up to them to do that parsing. But that's not a total solution: it seems plausible that a survey could be run with a structured date field that would not permit swapping the month and day _when the swapped date is invalid (month > 12)_ but would permit swaps that don't create invalid dates. And so on for other noise types. There is no way to achieve that with our approach.

With our current approach, the end result is that the column type is _always_ maximally broad to allow all noise types, whether or not those noise types are actually configured to happen.

It is both wasteful in terms of memory, and unrealistic, to say that all columns would be strings even when they're being recorded perfectly. That is true of DOB when it is configured to be perfect, just as much as it is true of survey_date which we have forced to be perfect.

I can easily imagine in the future that we might want to introduce some noise to this survey_date column -- it is metadata, and I'd expect it to be much less messy than the answers you get asking someone for their name, but there could still be administrative or clerical errors.

Note: a separate issue is the "unit" of error, which will be especially egregious in the case of metadata -- probably, if a survey_date were recorded wrong, everyone surveyed on that date (by a certain survey taker?) would have the same error. This is something we already ignore with e.g. addresses getting independent noise between members of the same household.